### ISSUE TITLE
'data' for bus: a 'tab' is lacking

### ISSUE THREAD

**Issue by Fresher14** [Toroid-io/vcd2wavedrom#5]
Proposal is to replace:
```
            if isbus:
                if lastval != j[1]:
                    digit = '='
                if 'x' not in j[1]:
                    drom['signal'][idromsig]['data'].append(
                            format(int(j[1], 2), 'X')
                    )
                else:
                    digit = 'x'

```
with:
```
            if isbus:
                if lastval != j[1]:
                    digit = '='
                    if 'x' not in j[1]:
                        drom['signal'][idromsig]['data'].append(
                                format(int(j[1], 2), 'X')
                        )
                    else:
                        digit = 'x'

```

**Comment by andresmanelli** [comment]
@Fresher14 Thank you very much for this ! Would you like to submit a PR?

**Comment by Fresher14** [comment]
PR submitted ! You can review it now.

**Comment by andresmanelli** [comment]
Fixed in #6
